site/docs/components/badge/accessibility.mdx

```diff
@@ -12,3 +12,7 @@ data:
 
 - When a badge is attached to a child component and that component receives focus, the focusable element should have an `aria-label` or `aria-labelledby` attribute that describes the badge, such as "9 notifications."
 - When the user navigates to an inline badge, the screen reader automatically announces the contents of the badge. Use `aria-label` to provide more context to the badge contents if needed.
+
+## Accessibility 2.1 disclaimer
+
+Please note that the dot badge indicator in dark mode falls slightly below the minimum contrast requirement of 3:1. This may impact the indicator visibility for users with visual impairments. This issue has been resolved in [Salt Next](https://www.saltdesignsystem.com/salt/theming/index#salt-next-theme).
```

Comment:
The link should be a relative one, probably start with `../theming` or `../../theming`

Reply:
done